### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by Robinnnnn**
### Summary of Changes

Autologging for debugging in development environment

### Demo or Before/After Pics

- If this is a new feature, include screenshots or recordings of the feature in action.
- If this PR makes visual changes, include before and after screenshots.

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
|  |  |

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Feed/FeedFilter.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { GalleryTouchableOpacity } from '../GalleryTouchableOpacity';
 import { Pill } from '../Pill';
 import { Typography } from '../Typography';
 
-export type ActiveFeed = 'Worldwide' | 'Following' | 'Profile' | 'Curated';
+export type ActiveFeed = 'Worldwide' | 'Following' | 'Profile' | 'For You';
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Feed/FeedFilter.tsx:9]: went ahead and changed these, was a bit weird to keep tracking `Curated`

File: apps/web/src/utils/logger.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { getTimestamp } from '~/shared/utils/time';
+
+export const _log: Console['log'] = (...args) => {
+  return console.log(`%c[${getTimestamp()}]`, 'color:#4878fa', `${args[0]}`, ...args.slice(1));
```
Comment (Robinnnnn) [inline apps/web/src/utils/logger.ts:4]: cool hack to add color in the browser

image

File: apps/mobile/src/components/Feed/FeedFilter.tsx (hunk 1 above)
Comment (kaitoo1) [reply apps/mobile/src/components/Feed/FeedFilter.tsx:9]: ohh nice thank you

File: apps/web/src/utils/logger.ts (hunk 2 above)
Comment (kaitoo1) [reply apps/web/src/utils/logger.ts:4]: woah

File: apps/mobile/src/contexts/MobileAnalyticsProvider.tsx [hunk 3]
```diff
@@ -20,13 +22,15 @@ if (token && apiUrl) {
 
 const track: TrackFunction = (eventName, eventProps) => {
   try {
+    console.log(`[${getTimestamp()}] Track`, eventName, eventProps);
```
Comment (kaitoo1) [inline apps/mobile/src/contexts/MobileAnalyticsProvider.tsx:25]: should we only track if local

File: apps/web/src/utils/logger.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { getTimestamp } from '~/shared/utils/time';
+
+export const _log: Console['log'] = (...args) => {
```
Comment (kaitoo1) [inline apps/web/src/utils/logger.ts:3]: same comment here should we only log if local

**Review by kaitoo1 — APPROVED** [review]

**Pull request merged**